torch/testing/_internal/opinfo/refs.py

```diff
@@ -113,6 +113,23 @@ def __init__(
         self.supports_nvfuser = supports_nvfuser
         assert isinstance(self.torch_opinfo, OpInfo)
 
+        def test_ref_export(name):
+            # make sure name is exported via __all__
+            import torch
+
+            lst = name.split(".")
+            mod_name = lst[:-1]
+            func_name = lst[-1]
+
+            mod = torch
+            for i in range(len(mod_name)):
+                mod = getattr(mod, mod_name[i])
```

Comment:
This is a hack. You want to do
```
mod = __import__(mod_name)
```
and remove the `import torch` above.

Reply:
i had that initially (also tried with importlib). these don't work for

```
    PythonRefInfo(
        "ops.nvprims.var_mean",
```

which is why i have the above.

alternatively, we could skip this module instead and use the proper import mechanism.

Reply:
@IvanYashchuk do you happen to know what's happening here?

Reply:
`torch.ops.nvprims.var_mean` (and all other `torch.ops` modules) is not created until `torch._prims` is imported.

Reply:
Also, it is not clear to me that this is the right place for this check. Why not making it a test in `test_ops`?

Reply:
i've just added a comment.
- importing `_prims` didn't work
- `__import__` doesn't work correctly with `__all__`: for a `torch` submodule, it just returns `torch` as the module
- `importlib.import_module` works, but not with `nvprims`, as explained above